src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountPayableSelectPage.tsx

```diff
@@ -22,19 +22,19 @@ type CardListItem = ListItem & {
 function QuickbooksCompanyCardExpenseAccountPayableSelectPage({policy}: WithPolicyConnectionsProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
-    const {accountsPayable} = policy?.connections?.quickbooksOnline?.data ?? {};
+    const {accountPayable} = policy?.connections?.quickbooksOnline?.data ?? {};
```

Comment:
Just a memo for myself. Checked this field name is correct

<img width="400" alt="Screenshot 2024-05-06 at 8 59 38 AM" src="https://github.com/Expensify/App/assets/98560306/1fbadbdd-f3f7-40ba-9bd8-68b4010ae112">